### PR TITLE
fix: Issues with Staking Earn

### PR DESCRIPTION
**Pull request by JChiaramonte7**

### CODE REVIEW

File: components/Table/Table.tsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ import Pagination from './Pagination';
 export type TablePalette = 'primary';
 
 const CARD_HEIGHT = '40px';
-const MAX_PAGE_ROWS = 10;
+const MAX_PAGE_ROWS = 12;
```
Comment (JChiaramonte7) [inline components/Table/Table.tsx:17]: There is a UI bug where react table pagination doesn't play nice with how we remove columns from the table if there's an `activeTab`. Would like to resolve this issue eventually, but going to increase max page rows for now.

File: sections/earn/IncentivesTable.tsx [hunk 2]
```diff
@@ -187,7 +187,7 @@ const IncentivesTable: FC<IncentivesTableProps> = ({ data, isLoaded, activeTab }
 				Header: <>{t('earn.incentives.options.rewards.title')}</>,
 				accessor: 'rewards',
 				Cell: (cellProps: CellProps<EarnItem, EarnItem['rewards']>) => {
-					const isDualRewards = cellProps.row.original.staked.asset === LP.UNISWAP_DHT;
+					const isDualRewards = typeof cellProps.row.original.rewards === 'object';
```
Comment (dvd-schwrtz) [inline sections/earn/IncentivesTable.tsx:190]: this doesn't feel like a good solution. can we do better?

File: components/Table/Table.tsx (hunk 1 above)
Comment (dvd-schwrtz) [reply components/Table/Table.tsx:17]: hmm... ok, let's add this to the list of issues to look into

File: sections/earn/IncentivesTable.tsx (hunk 2 above)
Comment (JChiaramonte7) [reply sections/earn/IncentivesTable.tsx]: yeah good point, fixed with something better

File: components/Table/Table.tsx (hunk 1 above)
Comment (JChiaramonte7) [reply components/Table/Table.tsx:17]: #521

**Review by clementbalestrat — APPROVED** [review]

**Comment by clementbalestrat** [comment]
LGTM @JChiaramonte7 can you please remove `package-lock.json` from your commit?

File: components/Table/Table.tsx (hunk 1 above)
Comment (justinjmoses) [reply components/Table/Table.tsx:17]: why even use pagination when we have scrolling?

**Review by justinjmoses — APPROVED** [review]

**Pull request merged**